dao/src/main/java/org/thingsboard/server/dao/device/DeviceProfileServiceImpl.java

```diff
@@ -159,7 +159,8 @@ public void deleteDeviceProfile(TenantId tenantId, DeviceProfileId deviceProfile
         this.removeDeviceProfile(tenantId, deviceProfile);
     }
 
-    private void removeDeviceProfile(TenantId tenantId, DeviceProfile deviceProfile) {
```

Comment:
Private have to stay private. Please, find the public methods to use with transactions (see comment on tenant for details)

Reply:
ok